lib/tpm-support-lib.robot

```diff
@@ -0,0 +1,58 @@
+*** Settings ***
+Documentation       Collection of keywords related to UEFI TPM support
+
+Library             ./tpm-support-lib.py
+Resource            ../keywords.robot
+Resource            ../keys.robot
+
+
+*** Keywords ***
+# TODO: Most options are split in 2 lines either because name is too long or
+#    they have too many arguments. Get TCG2 Menu Construction currently
+#    returns those as different list items
```

Comment:
@m-iwanicki I think there could be more submenus where multilines happenes, do we have some idea how to use these keywords in other then TCG2 menus?

Reply:
@TomaszAIR Keyword should probably work for many other menus with minimal changes.
@{additional_remove} should be argument or contain all possible values to be removed for every menu (there are not that many).
Number of pages could be determined by presence of down arrow

Reply:
And this is something worth to left here as TODO. Added here https://github.com/Dasharo/open-source-firmware-validation/pull/164/commits/7dbf68a4e47fbe29973912d42dc78c4d6aa31249